c_src/cmake/FindGFComplete.cmake

```diff
@@ -0,0 +1,33 @@
+include(ExternalProject)
+
+# get an uppercase version of the build type, for extracting build_type specific flags
+if(CMAKE_BUILD_TYPE)
+  string(TOUPPER ${CMAKE_BUILD_TYPE} BUILD_TYPE_UC)
+endif()
+
+ExternalProject_Add(gf-complete
+  PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/gf-complete
+  GIT_REPOSITORY    https://github.com/ceph/gf-complete.git
+  GIT_TAG           a6862d10c9db467148f20eef2c6445ac9afd94d8
+  UPDATE_COMMAND    ""
+  BUILD_IN_SOURCE   1
+  CONFIGURE_COMMAND autoreconf --install > /dev/null 2>&1 || autoreconf &&
+                    ./configure
+                    --prefix=${CMAKE_CURRENT_BINARY_DIR}
+                    --with-pic
+                    --disable-shared
+                    $ENV{CONFIGURE_ARGS}
+                    CC=${CMAKE_C_COMPILER}
+                    CFLAGS=${CMAKE_C_FLAGS_${BUILD_TYPE_UC}}
+  BUILD_COMMAND     make -j
+  BUILD_BYPRODUCTS  ${CMAKE_CURRENT_BINARY_DIR}/lib/libgf_complete.a
+  INSTALL_COMMAND   make install
+  )
```

Comment:
The mixing of Autotools with CMake here makes me nervous. I don't want any environment or make variables passed down from `miner.mk` to get clobbered. I guess I'll find out after this merges. If the miner build for the Helium Hotspots breaks we can fix it then.

Reply:
From the best that I can tell it's fine, but I can't say with 100% confidence that it can't cause problems.